main.go

```diff
@@ -120,6 +120,10 @@ var cmdInstall = cli.Command{
 			Name:  "no-check",
 			Usage: "Skip pre-install tests.",
 		},
+		cli.BoolFlag{
+			Name:  "downgrade",
```

Comment:
`--downgrade-ok` or `--allow-downgrade`. When I see `--downgrade`, I think that downgrades are the _only_ thing we allow.